### PR TITLE
Added GATT Barometric Pressure Trend Characteristic

### LINKED ISSUE #58: Implement standard GATT Characteristics
Implement the [standard GATT Characteristics](https://www.bluetooth.com/specifications/gatt/characteristics) in `GATTCharacteristic.swift` by implementing the `GATTProfileCharacteristic` protocol.

### PR DESCRIPTION
**Pull request by carlos21**
**Issue**

Fixes #58.

**What does this PR Do?**

Added GATT Barometric Pressure Trend characteristic.

**Where should the reviewer start?**

GATTCharacteristics.swift

### CODE REVIEW

File: Sources/GATTCharacteristic.swift [hunk 1]
```diff
@@ -1841,6 +1841,173 @@ extension GATTAerobicThreshold: CustomStringConvertible {
...
+    }
+}
+
+public enum GATTBarometricPressureTrend: Unitless, GATTProfileCharacteristic {
+    
+    internal static let length = MemoryLayout<UInt8>.size
+    
+    public static var uuid: BluetoothUUID { return .barometricPressureTrend }
+    
+    case unknown = 0x00
```
Comment (colemancda) [inline Sources/GATTCharacteristic.swift]: Add documentation for members / cases

File: Sources/GATTCharacteristic.swift [hunk 2]
```diff
@@ -1841,6 +1841,173 @@ extension GATTAerobicThreshold: CustomStringConvertible {
...
+    }
+    
+    public var data: Data {
+        
+        return Data([rawValue])
+    }
+    
+}
+
+public struct Unitless: BluetoothUnit {
```
Comment (colemancda) [inline Sources/GATTCharacteristic.swift]: Remove

File: Tests/BluetoothTests/GATTCharacteristicTests.swift [hunk 3]
```diff
@@ -339,4 +341,63 @@ final class GATTCharacteristicTests: XCTestCase {
...
+        
+        guard let characteristic = GATTAnaerobicHeartRateUpperLimit(data: data)
+            else { XCTFail("Could not decode from bytes"); return }
+        
+        XCTAssertEqual(characteristic.data, data)
+        XCTAssertEqual(characteristic.beats, beats)
+        XCTAssertEqual(characteristic.description, "34")
+        XCTAssertEqual(beats.description, "34")
+        XCTAssertEqual(GATTAnaerobicHeartRateUpperLimit.uuid, .anaerobicHeartRateUpperLimit)
+        XCTAssertEqual(BeatsPerMinute.unitType, .beatsPerMinute)
```
Comment (JurjDev) [inline Tests/BluetoothTests/GATTCharacteristicTests.swift]: You are missing a Equatable test of GATTAnaerobicHeartRateUpperLimit

File: Tests/BluetoothTests/GATTCharacteristicTests.swift [hunk 4]
```diff
@@ -339,4 +341,63 @@ final class GATTCharacteristicTests: XCTestCase {
...
+        guard let characteristic = GATTAnaerobicHeartRateLowerLimit(data: data)
+            else { XCTFail("Could not decode from bytes"); return }
+        
+        XCTAssertEqual(characteristic.data, data)
+        XCTAssertEqual(characteristic.beats, beats)
+        XCTAssertEqual(characteristic.description, "34")
+        XCTAssertEqual(beats.description, "34")
+        XCTAssertEqual(GATTAnaerobicHeartRateLowerLimit.uuid, .anaerobicHeartRateLowerLimit)
+        XCTAssertEqual(BeatsPerMinute.unitType, .beatsPerMinute)
+    }
```
Comment (JurjDev) [inline Tests/BluetoothTests/GATTCharacteristicTests.swift]: You are missing a Equatable test of GATTAnaerobicHeartRateLowerLimit

Comment (colemancda) [reply Tests/BluetoothTests/GATTCharacteristicTests.swift]: Add `XCTAssertNotEqual()` as well

**Comment by colemancda** [comment]
Result of [Integration 1](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/4879f475f08d399f3fa7c50903f00258/4879f475f08d399f3fa7c50903f00524)
---
*Duration*: 10 minutes and 24 seconds
*Result*: **Perfect build!** All 165 tests passed. :+1:
*Test Coverage*: 86%

File: Sources/GATTCharacteristic.swift [hunk 5]
```diff
@@ -1842,6 +1842,27 @@ extension GATTAerobicThreshold: CustomStringConvertible {
 }
 
 /**
+<<<<<<< HEAD
```
Comment (colemancda) [inline Sources/GATTCharacteristic.swift]: Merge artifact must be resolved and removed

File: Sources/GATTCharacteristic.swift [hunk 6]
```diff
@@ -1859,18 +1880,102 @@ public struct GATTAnalog: GATTProfileCharacteristic {
...
         self.analog = analog
+>>>>>>> master
     }
     
     public init?(data: Data) {
         
         guard data.count == type(of: self).length
             else { return nil }
         
+<<<<<<< HEAD
```
Comment (colemancda) [inline Sources/GATTCharacteristic.swift]: Merge artifact must be resolved and removed

File: Sources/GATTCharacteristic.swift [hunk 7]
```diff
@@ -1859,18 +1880,102 @@ public struct GATTAnalog: GATTProfileCharacteristic {
...
+        return lhs.beats == rhs.beats
+    }
+}
+
+extension GATTAnaerobicHeartRateUpperLimit: CustomStringConvertible {
+    
+    public var description: String {
+        
+        return beats.description
+=======
```
Comment (colemancda) [inline Sources/GATTCharacteristic.swift]: Merge artifact must be resolved and removed

**Comment by colemancda** [comment]
Result of [Integration 2](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/4879f475f08d399f3fa7c50903f00258/054202d02a0d1e038383e0e5944026b5)
---
*Duration*: 56 seconds
*Result*: **26 errors, failing state: build-errors**

File: Sources/GATTCharacteristic.swift (hunk 5 above)
Comment (colemancda) [reply Sources/GATTCharacteristic.swift]: @carlos21

**Comment by colemancda** [comment]
Result of [Integration 2](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/4879f475f08d399f3fa7c50903f00258/054202d02a0d1e038383e0e5944026b5)
---
*Duration*: 56 seconds
*Result*: **26 errors, failing state: build-errors**

**Comment by colemancda** [comment]
Result of [Integration 2](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/4879f475f08d399f3fa7c50903f00258/054202d02a0d1e038383e0e5944026b5)
---
*Duration*: 56 seconds
*Result*: **26 errors, failing state: build-errors**

**Comment by colemancda** [comment]
Result of [Integration 3](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/4879f475f08d399f3fa7c50903f00258/054202d02a0d1e038383e0e59449d82c)
---
*Duration*: 4 minutes and 46 seconds
*Result*: **Perfect build!** All 168 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 3](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/4879f475f08d399f3fa7c50903f00258/054202d02a0d1e038383e0e59449d82c)
---
*Duration*: 4 minutes and 46 seconds
*Result*: **Perfect build!** All 168 tests passed. :+1:
*Test Coverage*: 86%

**Comment by colemancda** [comment]
Result of [Integration 3](https://stlt.herokuapp.com/v1/xcs_deeplink/Colemans-MacBook-Pro.local/4879f475f08d399f3fa7c50903f00258/054202d02a0d1e038383e0e59449d82c)
---
*Duration*: 4 minutes and 46 seconds
*Result*: **Perfect build!** All 168 tests passed. :+1:
*Test Coverage*: 86%

**Pull request merged**